<launch>

  <arg name="simulated" default='true'/>


  <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch"/>

  <node name="data_server" pkg="cobots_core" type="data_server.py">
    <!--TODO-->
  </node>

  <node name="robot_control" pkg="cobots_core" type="robot_control.py">
    <!--TODO-->
  </node>

  <node name="application_tf_handler" pkg="cobots_core" type="application_tf_handler.py">
    <!--TODO-->
  </node>

</launch>
